contrib/README.md

```diff
@@ -11,39 +11,66 @@ A Dockerfile for Fedora can be generated in `contrib`.
 To prepare the Docker container run this command:
 
 ```
-OS=fedora ./generate_docker.py
+OS=fedora ./generate_docker.py build
 ```
 
 To build the RPMs run this command (from the root of your git checkout):
 
 ```
-docker run --privileged -t -v `pwd`:/build fwupd-fedora
+docker run --privileged -t -v `pwd`:/github/workspace fwupd-fedora
 ```
 
 RPMs will be made available in your working directory when complete.
 
+To build additional RPM packages for Qubes OS (fwupd-qubes-dom0 and
+fwupd-qubes-vm) prepare Qubes specific docker container:
+
+```
+OS=fedora-qubes ./generate_docker.py build
+```
+
+Build RPMs with following command:
+
+```
+docker run --privileged -t -v `pwd`:/github/workspace fwupd-fedora-qubes
+```
+
+
 ## DEB packages
 A Dockerfile for Debian or Ubuntu can be generated in `contrib`.
 
 To prepare the Docker container run one of these commands:
 
 ```
-OS=debian-x86_64 ./generate_docker.py
-OS=debian-i386 ./generate_docker.py
-OS=ubuntu-x86_64 ./generate_docker.py
+OS=debian-x86_64 ./generate_docker.py build
+OS=debian-i386 ./generate_docker.py build
+OS=ubuntu-x86_64 ./generate_docker.py build
 ```
 
 
 To build the DEBs run one of these commands (from the root of your git checkout):
 
 ```
-docker run --privileged -t -v `pwd`:/build fwupd-debian-x86_64
-docker run --privileged -t -v `pwd`:/build fwupd-debian-i386
-docker run --privileged -t -v `pwd`:/build fwupd-ubuntu-x86_64
+docker run --privileged -t -v `pwd`:/github/workspace fwupd-debian-x86_64
+docker run --privileged -t -v `pwd`:/github/workspace fwupd-debian-i386
+docker run --privileged -t -v `pwd`:/github/workspace fwupd-ubuntu-x86_64
```

Comment:
were these wrong?  If so, please split this to a separate commit

Reply:
Done.